packages/web-app-files/src/components/FilesList/ResourceDetails.vue

```diff
@@ -39,6 +42,25 @@ export default defineComponent({
       required: false,
       default: null
     }
+  },
+  setup(props) {
+    const { getDefaultAction, triggerDefaultAction } = useFileActions()
+    const fileActionsDownloadFile = useFileActionsDownloadFile()
+    const openWithDefaultAppQuery = useRouteQuery('openWithDefaultApp')
+
+    const hasDefaultAction =
+      getDefaultAction({
```

Comment:
You could implement a `getDefaultEditorAction` method in the `useFileActions` composable and use it here. That would be a little less `meh` then. Or add an option to the `getDefaultActions` options param for omitting system actions. That would be fine as well.

Reply:
done